spec/avram/queryable_spec.cr

```diff
@@ -856,6 +856,95 @@ describe Avram::Queryable do
     end
   end
 
+  describe "#group_count" do
+    context "when no records exist" do
+      it "0 grouped columns returns { [] => 0 }" do
+        query = UserQuery.new
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
+
+      it "1 grouped columns returns { [] => 0 }" do # TODO: What should we do here?
+        query = UserQuery.new.group(&.name)
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
+
+      it "2 grouped columns returns { [] => 0 }" do # TODO: What should we do here?
+        query = UserQuery.new.group(&.name)
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
```

Comment:
These 2 fail because the query returns an entirely empty `Hash`

Reply:
![image](https://user-images.githubusercontent.com/6721355/147843627-d88350b9-7047-44dc-ba48-85915bda5663.png)